conjur/user/user_controller.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+"""
+UserController module
+
+This module is the controller that facilitates all user actions
+"""
+
+# Builtins
+import getpass
+import logging
+import sys
+import requests
+
+# Internals
+from conjur.errors import InvalidPasswordComplexity, InvalidOperation
+
+
+class UserController():
+    """
+    UserController
+
+    This class represents the Presentation Layer for the User command.
+    """
+    password_complexity_constraints = "The password must contain at least 12 characters: " \
```

Comment:
SHOULD be capital case letters because it is const

Reply:
Also it belongs to class and not to instance itself so maybe use it without anything or with `UserController.PASSWORD_COMPLEXITY_CONSTRAINS`

Reply:
Also maybe adding the word message to the names can help but this is up to you to decide

Reply:
I actually think it would be better in constants
moving, capitalizing,  and adding message to the end